### DISCUSSION TITLE
Unable to install Exo Community Edition 6.4

### DISCUSSION THREAD

**Discussion by ahmedqat** [exoplatform/exo-community#43] [Q&A]
Hello All, first time posting for help online honestly.

So I attempted to install Exo Community Edition 6.4  on my ubuntu VM that is running on virtual box. 

I am running ubuntu server 22.04.3. My VM is set up so that I have 2 adapters, one NAT adapter and one host-only adapter. This allows me to connect to the host through ssh.

I installed Exo using docker compose following the installation instructions on this website; https://docs.exoplatform.org/guide/getting-started/start-community.html#system-requirements

After installing and setting everything up, attempting to access IP:80 gives me a 502 bad gateway error. 

I would gladly appreciate your help in assisting me with this matter and I am willing to prevent any information if you require it.

**Comment by rdenarie** [comment]
Hi @ahmedqat 
Can you provide the startup log of the docker-compose command ?

Thanks

**Reply by ahmedqat** [reply]
Hi @rdenarie .
Thank you for your reply.

Actually I have managed to solve the issue and everything is working fine now. Turns out elastic search kept on crashing for me, after doing some research, I changed the vm.max_map_count to 262144 and the program was able to run flawlessly.

**Comment by rdenarie** [comment]
Perfect, thanks for the update
